src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -70,8 +70,10 @@ const logger = createLogger("Order/Routes/Shipping/index.tsx")
 export class ShippingRoute extends Component<ShippingProps, ShippingState> {
   state: ShippingState = {
     shippingOption: ((this.props.order.requestedFulfillment &&
-      this.props.order.requestedFulfillment.__typename.toUpperCase()) ||
-      "SHIP") as OrderFulfillmentType,
+      (this.props.order.requestedFulfillment.__typename === "CommerceShip"
```

Comment:
is there a way we could make this little more readable? having hard time reading this line's logic

Reply:
Just updated it!